code/modules/racial_roundstartperks/racial_perk.dm

```diff
@@ -0,0 +1,253 @@
+///////////////////////////////////////Akula Based perks
+/datum/perk/recklessrage
+	name = "Reckless Rage"
+	desc = "Your body is powerful and strong when you succumb to instinct, but doing so leaves you without much higher reasoning for a short time. The rush of chemicals is also highly addictive \
+	and often times will leave your body weaker for a short time..... and such reckless rage makes you have a extreme tunnel vision and bright lights are extremely effective againts you. Such is everything else really. You are tough but weak."
+	active = FALSE
+	passivePerk = FALSE
+	var/init_sight
+	var/init_flash
+	var/obj/screen/lightOverlay = null
+
+/datum/perk/recklessrage/assign(mob/living/carbon/human/H)
+	..()
+	init_sight = holder.additional_darksight
+	init_flash = holder.flash_mod
+	holder.additional_darksight = -3
+	holder.flash_mod += 4
+
+/datum/perk/recklessrage/remove()
+	holder.additional_darksight = init_sight
+	holder.flash_mod = init_flash
+	..()
+/datum/perk/recklessrage/assign(mob/living/carbon/human/H)
+	..()
+	holder.brute_mod_perk += 0.10
+	holder.burn_mod_perk += 0.10
+	holder.oxy_mod_perk += 0.10
+	holder.toxin_mod_perk += 0.10
+	holder.stats.changeStat(STAT_COG, -2)
+	holder.stats.changeStat(STAT_ANA, 1)
+	holder.stats.changeStat(STAT_BIO, -2)
+
+/datum/perk/recklessrage/remove()
+	holder.brute_mod_perk -= 0.10
+	holder.burn_mod_perk -= 0.10
+	holder.oxy_mod_perk -= 0.10
+	holder.toxin_mod_perk -= 0.10
+	holder.stats.changeStat(STAT_COG, 2)
+	holder.stats.changeStat(STAT_ANA, -1)
+	holder.stats.changeStat(STAT_BIO, 2)
+	..()
+
+
+/datum/perk/recklessrage/activate()
+	var/mob/living/carbon/human/user = usr
+	if(!istype(user))
+		return ..()
+	if(world.time < cooldown_time)
+		to_chat(usr, SPAN_NOTICE("Your body has been taxed to its limits, you need more time to recover before using this ability again."))
+		return FALSE
+	cooldown_time = world.time + 10 MINUTES
+	user.visible_message("<b><font color='red'>[user] lets out deep guttural growl as their eyes glaze over!</font><b>", "<b><font size='3px'><font color='red'>You abandon all reason as your sink into a blood thirsty frenzy!</font><b>", "<b><font color='red'>You hear a terrifying roar!</font><b>")
+	playsound(usr.loc, 'sound/voice/akularoar.ogg', 50, 1)
+	log_and_message_admins("used their [src] perk.")
+	user.reagents.add_reagent("furiadol", 5)
+	return ..()
+
+/datum/reagent/stim/furiadol
+	name = "Furiadol"
+	id = "furiadol"
+	description = "A chemical naturally produced by several species that sends them into an all consuming frenzy."
+	taste_description = "bitterness"
+	reagent_state = LIQUID
+	color = "#5f95e2"
+	nerve_system_accumulations = 105 // so people can't spam it with others perks
```

Comment:
```suggestion
	nerve_system_accumulations = 70 // Minimum baseline NSA for certain races/backgrounds is 80~85, this is also in par with MENACE
```
If you set it to 100 or higher any race will instantly throw up of maximum NSA threshold, granted this should still make anyone vomit if they have fucked up nerves but better than outright inducing NSA break.